Prox/Prox/Data/EventsController.swift

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+protocol EventsControllerDelegate: class {
+    func eventController(_ eventController: EventsController, didUpdateEvents: [Event])
+    func eventController(_ eventController: EventsController, didError error: Error)
+}
+
+class EventsController {
```

Comment:
This will become clearer with future patches as I start sending notifications. Will rename to `EventNotificationManager` to make it clearer.